### PR TITLE
Add topic "CSS Framework"

### PR DESCRIPTION
**Pull request by shaedrich**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I am not the sole author or employee of a company who created the topic or collection I'm changing.

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

In web development, there's not really a way around CSS frameworks nowadays.

### Something that does not neatly fit into the binary options above

- [x] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/css-framework/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+display_name: CSS Framework
+short_description: CSS Frameworks usually offer one or more stylesheets with basic styling/generic components, and many of them use preprocessors.
+topic: css-framework
+related: css, bootstrap, tailwind, zurb-foundation
+wikipedia_url: https://en.wikipedia.org/wiki/CSS_framework
+---
+**CSS Frameworks** offer one or multiple stylesheets from basic styling to generic components to some extend, including but not limited to
```
Comment (octokatherine) [inline topics/css-framework/index.md:8]:
```suggestion
**CSS Frameworks** offer one or multiple stylesheets from basic styling to generic components to some extend, including but not limited to
```

Can we iterate on the wording here, I'm not sure I understand "to some extend"

File: topics/css-framework/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+* Reboot (aka "Preflight")
+* Layout (often times "Grid")
+* Forms, Tables, etc.
+* [Typography](https://github.com/topics/typography)
+* [Responsiveness](https://github.com/topics/responsive) (often times including breakpoints and [viewport](https://github.com/topics/viewport))
+
+They often times have their own class notation and many of them use [preprocessors](https://github.com/topics/css-preprocessor) like [Sass](https://github.com/topics/sass), [Less](https://github.com/topics/less) oder [Stylus](https://github.com/topics/stylus).
```
Comment (octokatherine) [inline topics/css-framework/index.md:15]:
```suggestion
They often have their own class notation and many use [preprocessors](https://github.com/topics/css-preprocessor) like [Sass](https://github.com/topics/sass), [Less](https://github.com/topics/less) or [Stylus](https://github.com/topics/stylus).
```

Small suggestions on rewording and a fix for what I believe is a typo

File: topics/css-framework/index.md (hunk 1 above)
Comment (shaedrich) [reply topics/css-framework/index.md:8]: It's a typo. I actually intended to write "exten**t**"

File: topics/css-framework/index.md (hunk 2 above)
Comment (shaedrich) [reply topics/css-framework/index.md:15]: It is&mdash;embarrassing German auto correct 😅

**Review by kenyonj — APPROVED** [review]
🥇 🙇🏻‍♂️

**Pull request merged**